include/tcpdf_cmssignature.php

```diff
@@ -0,0 +1,545 @@
+<?php
+/*
+// File name   : tcpdf_cmssignature.php
+// Version     : 1.0.1
+// Begin       : 2023-05-25
+// Last Update : 2023-05-25
+// Author      : Hida - https://github.com/hidasw
+// License     : GNU GPLv3
+*/
+/**
+ * @class tcpdf_cms_signature
+ * Manage CMS Signature for TCPDF.
+ * @version 1.0.001
+ * @author M Hida
+ */
+class tcpdf_cms_signature {
```

Comment:
can this file have less code ?

Reply:
Seems hard. It's better if there's another FOSS library we can use to append an existing PKCS7 structure or employ other methods, instead of parsing and appending PKCS7 data standalone. However, they are usually intended for general purposes with many functions that we don't even need here.